sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -204,6 +218,28 @@ def _create(self, resource_type, content_settings=None, metadata=None, **kwargs)
             For example, if p is 0777 and u is 0057, then the resulting permission is 0720.
             The default permission is 0777 for a directory and 0666 for a file. The default umask is 0027.
             The umask must be specified in 4-digit octal notation (e.g. 0766).
+        :keyword str owner:
+            The owner of the blob or directory. Default value is None.
+        :keyword str group:
+            The owning group of the blob or directory. Default value is None.
+        :keyword str acl:
+            Sets POSIX access control rights on files and directories. The value is a
+            comma-separated list of access control entries. Each access control entry (ACE) consists of a
+            scope, a type, a user or group identifier, and permissions in the format
+            "[scope:][type]:[id]:[permissions]". Default value is None.
+        :keyword str lease_id:
```

Comment:
Let's copy the descriptions in `acquire_lease` for `lease_id` and `lease_duration`. In general, I find the generated descriptions are not always great and I usually try and find the same param somewhere else and copy that docstring if it sounds good or at least for consistency.

"Proposed lease ID, in a GUID string format. The DataLake service returns
400 (Invalid request) if the proposed lease ID is not in the correct format."

"Specifies the duration of the lease, in seconds, or negative one
(-1) for a lease that never expires. A non-infinite lease can be
between 15 and 60 seconds. A lease duration cannot be changed
using renew or change."